### DISCUSSION TITLE
My Extension to sync all settings and Extensions among all VS Code based IDEs(including VS Code) doesn't install anything and just shows a message "All extensions are already installed!" (Help me fix this ASAP)

### DISCUSSION THREAD

**Discussion by anugrahjames-official** [microsoft/vscode-discussions#2433] [Extension Development QnA]
Link to my Extension sourcecode : https://github.com/anugrahjames-official/vscode-sync-extension

![Screenshot From 2024-12-07 04-10-20](https://github.com/user-attachments/assets/24eaab0b-d9ce-443c-ad79-3c1a2af86b7d)

**Comment by olguzzar** [comment]
Hi @anugrahjames-official, are you still having this issue? If so, can you share some of the debugging steps you've taken so far?

**Reply by anugrahjames-official** [reply]
Yes, I still have this issue😵‍💫, I tried to use different ways like using config files for installing and other OS specific ways to install extensions but nothing works so far
